packages/google_maps_flutter/google_maps_flutter_android/pubspec.yaml

```diff
@@ -2,11 +2,11 @@ name: google_maps_flutter_android
 description: Android implementation of the google_maps_flutter plugin.
 repository: https://github.com/flutter/plugins/tree/main/packages/google_maps_flutter/google_maps_flutter_android
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+maps%22
-version: 2.1.10
+version: 2.2.0
 
 environment:
   sdk: ">=2.14.0 <3.0.0"
-  flutter: ">=2.8.0"
+  flutter: ">=3.0.0"
```

Comment:
Just out of curiosity, would this also cause the `downgrade analyze` test to fail for `google_maps_flutter` package. It would make the minimum Flutter version it could use `3.0.0` on Android, but could still use `2.8.0` for iOS. 

https://github.com/flutter/plugins/blob/main/packages/google_maps_flutter/google_maps_flutter/pubspec.yaml#L9

Reply:
Nope, the resolver just won't pick this version if you're on an older version of Flutter. Updating the minimum version of the implementation package required by the app-facing package is what would require us to update the app-facing min version.

This is a small perk of federation; for now people can get updates to anything but the Android implementation even on old versions.